front/app/containers/Admin/pagesAndMenu/containers/EditHomepage/index.tsx

```diff
@@ -155,7 +152,6 @@ const EditHomepage = ({ intl: { formatMessage } }: InjectedIntlProps) => {
                 editLinkPath={linkToPath}
                 titleMessageDescriptor={titleMessageDescriptor}
                 tooltipMessageDescriptor={tooltipMessageDescriptor}
```

Comment:
I think disabling individual toggles via `isLoading` is still useful for any toggle in the pending state, otherwise there's no feedback until the call finishes, which could be a moment on a slower connection:

https://user-images.githubusercontent.com/3614128/196670949-ae936192-bb10-4a4b-b28e-6d85b12bb8fd.mov